{"name": "test_check_wrong_password", "status": "broken", "statusDetails": {"message": "TypeError: 'TextBox' object is not callable", "trace": "self = <user_settings_tests.TestUserSettings object at 0x10c6d7710>\n\n    def test_check_wrong_password(self):\n        self.settings_page.get_settings_side_menu_button().click()\n        self.settings_page.get_change_email_button().click()\n>       self.fill_email_and_password_change_email(\"fedorchuck_mykhailo@gmail.com\", \"Artsem1997\")\n\nuser_settings_tests.py:219: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <user_settings_tests.TestUserSettings object at 0x10c6d7710>, email = 'fedorchuck_mykhailo@gmail.com'\npassword = 'Artsem1997'\n\n    @allure.step(\"Data to fill in email and password fields in 'Change email'\")\n    def fill_email_and_password_change_email(self, email, password):\n>       self.settings_page._new_email_address_field().fill_field(email)\nE       TypeError: 'TextBox' object is not callable\n\nuser_settings_tests.py:213: TypeError"}, "steps": [{"name": "Data to fill in email and password fields in 'Change email'", "status": "broken", "statusDetails": {"message": "TypeError: 'TextBox' object is not callable\n", "trace": "  File \"/Users/pauliukevi4i/Stotska_final_project/venv/lib/python3.11/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/Users/pauliukevi4i/Stotska_final_project/user_settings_tests.py\", line 213, in fill_email_and_password_change_email\n    self.settings_page._new_email_address_field().fill_field(email)\n    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n"}, "parameters": [{"name": "email", "value": "'fedorchuck_mykhailo@gmail.com'"}, {"name": "password", "value": "'Artsem1997'"}], "start": 1690336050029, "stop": 1690336050029}], "start": 1690336048746, "stop": 1690336050030, "uuid": "31a93cf8-6282-4196-a044-8c2325528fc2", "historyId": "ae558438c68db70490375a4539641024", "testCaseId": "ae558438c68db70490375a4539641024", "fullName": "user_settings_tests.TestUserSettings#test_check_wrong_password", "labels": [{"name": "suite", "value": "user_settings_tests"}, {"name": "subSuite", "value": "TestUserSettings"}, {"name": "host", "value": "AirPauliukevich"}, {"name": "thread", "value": "8657-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "user_settings_tests"}]}